lesson_06/task06_2/client/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bufio"   // buffers for reading
+	"context" // pkg that allows us to set timeout
+	"fmt"     // output into terminal
+	"log"     // logging pkg
+	"net"     // main pkg for networking
+	"os"
+	"time" // pkg for time constant
+)
+
+func main() {
+	// create a dialer
+	var d net.Dialer
+	fmt.Print("Enter \"exit\" to terminate the program!\n")
+	// loop for entering messages (ends at input "exit")
+	for {
+		// taking our message from the console
+		fmt.Print("Enter your message: ")
+		message, _ := bufio.NewReader(os.Stdin).ReadString('\n')
+
+		if message == "exit\n" {
+			return
+		}
+
+		// server port number
+		const port = 8081
+
+		// message[:len(message) - 1] - removes '\n' for logging
+		fmt.Printf("Sending message: %s; to port: %d\n", message[:len(message)-1], port)
```

Comment:
also good approach to define string constants (for ex. some log literals or error messages) 
I recommend to move such strings into constants